ondemand-testandpush-metrics.py

```diff
@@ -79,7 +79,12 @@ def parse_test(line):
         return 0
     seconds = line[idx + len(marker1):line.find(marker2, idx)]
     if seconds:
-        return float(seconds)
+        if ',' in seconds:
+            seconds = seconds.replace(',', '.')
```

Comment:
I don't think this is correct. `933,915.666` means `933915.666` so you should just remove the `,`, not replace it with a `.`.

Reply:
fixed in https://github.com/nuxeo/integration-scripts/pull/30
review in progress